src/latexify/ast_utils_test.py

```diff
@@ -146,3 +146,21 @@ def test_extract_int_invalid() -> None:
         ast_utils.extract_int(ast_utils.make_constant("123"))
     with pytest.raises(ValueError, match=r"^Unsupported node to extract int"):
         ast_utils.extract_int(ast_utils.make_constant(b"123"))
+
+
+@pytest.mark.parametrize(
+    "value,expected",
+    [
+        (ast.Call(ast.Name("hypot"), ast.Constant("foo")), "hypot"),
+        (
+            ast.Call(
+                ast.Attribute(ast.Name("math"), "hypot", ast.Load()),
+                ast.Constant("foo"),
+            ),
+            "hypot",
+        ),
+        (ast.Call(ast.Constant(123), ast.Constant("foo")), None),
```

Comment:
- These `Call`s are illegal because the `args` parameter should take a list of `expr`.
- The `func` parameter could take any `expr`, but in this case the constant `123` is never populated here in the actual AST. I guess some other example is preferred, e.g., `ast.Call(func=ast.Name("foo"), args=[])`, which represents `foo()(...)`